lib/engineering/analysis/footing/centric_isolated.rb

```diff
@@ -0,0 +1,50 @@
+require 'errors/engineering/analysis/section_direction_error'
+
+module Engineering
+  module Analysis
+    module Footing
+      class CentricIsolated
+        ORTHOGONALITIES = %i[length_1 length_2].freeze
+
+        def initialize(footing:, load_from_column:, section_direction:)
+          if ORTHOGONALITIES.none?(section_direction)
+            raise Engineering::Analysis::SectionDirectionError.new(section_direction, ORTHOGONALITIES)
+          end
+
+          @footing = footing
+          @load_from_column = load_from_column
+          @section_direction = section_direction
+        end
+
+        def solicitation_load
+          solicitation * @footing.public_send(orthogonal_direction)
+        end
+
+        def max_shear_solicitation
+          solicitation_load * @footing.public_send(@section_direction)
+        end
+
+        def shear_solicitation_at(distance_from_footing_center:)
+          footing_section_length = @footing.public_send(@section_direction)
+
+          solicitation_load * (footing_section_length - distance_from_footing_center)
+        end
+
+        def bending_solicitation
+          0.25 * max_shear_solicitation * @footing.public_send(@section_direction)
+        end
+
+        private
+
+        def solicitation
+          @load_from_column.value / @footing.horizontal_area
+        end
+
+        def orthogonal_direction
+          orthogonal = ORTHOGONALITIES - [@cut_direction]
```

Comment:
What is `@cut_direction`? Should it be `@section_direction` instead?
```suggestion
          orthogonal = ORTHOGONALITIES - [@section_direction]
```